pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
+spec:
+  containers:
+  - name: barepod
+  # Docker image name
+    image: 172.30.223.233:5000/sample-project/batch-cgs
```

Comment:
When you build the script in OpenShift, OS create image name with Ip address. That's how i am able to run the script that is built in OS. This might be different in Production, but i don't how that is configured with our IP address